modules/argocd-installing-argocd.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/using-argocd.adoc
+
+[id="argocd-installing-argocd_{context}"]
+= Installing ArgoCD
+
+To use ArgoCD to manage your cluster and application configuration files, you must install it after you install your {product-title} cluster.
+
+[NOTE]
+====
+These steps will be deprecated when the ArgoCD Operator is added to OperatorHub.
+====
+
+.Prerequisites
+
+* You installed a {product-title} cluster and the `oc` command line tool.
+* You installed the `oc` command line tool.
+* You have the link:https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/[kubeconfig] file for your cluster. The default `kubeconfig` file is in the `~/.kube/config` folder on the machine that you installed your cluster from.
+
+.Procedure
+
+. Create a new project for ArgoCD:
++
+----
+$ oc new-project argocd
+----
+
+. Apply the ArgoCD manifest file to your cluster:
++
+----
+$ oc apply -n argocd -f https://raw.githubusercontent.com/argoproj/argo-cd/stable/manifests/install.yaml
```

Comment:
@dgoodwin should this refer to a specific version of ArgoCD that we have verified like:
https://raw.githubusercontent.com/argoproj/argo-cd/release-1.4/manifests/install.yaml

Reply:
I am not sure, we run the risk of pointing people to an abandoned ArgoCD version.

Is the operator ready to go enough that we could update the docs to install via the operator?

Reply:
@dgoodwin Using latest has the risk of some of the instructions might change in the latest release compared to the version we have verified. 

Regarding the operator, we could change already but I prefer to update in the next iterations for 4.4  to have the operator mature a bit more.